libs/sdk-ui-filters/src/AttributeFilter/AttributeFilterButton.tsx

```diff
@@ -360,7 +418,7 @@ export const AttributeFilterButtonCore: React.FC<IAttributeFilterButtonProps> =
                 }
                 body={
                     <AttributeDropdownBody
-                        items={filterOptions}
+                        items={validElements?.items ?? []}
                         totalCount={totalCount}
                         selectedItems={selectedFilterOptions}
```

Comment:
When you provide a placeholder with attribute filter with preselected elements, is it correctly handled and preselected here?